schemas/configuration/v5/server_config.json.in

```diff
@@ -6,38 +6,89 @@
         "advanced_settings": {
             "type": "object",
             "properties": {
-                "default_number_of_transfer_threads": {"type": "integer"},
-                "default_temporary_password_lifetime_in_seconds": {"type": "integer"},
+                "default_number_of_transfer_threads": {
+                    "type": "integer",
+                    "minimum": 1
+                },
```

Comment:
I believe 0 forces streaming as well. Not sure if we want to allow that or not.

Reply:
Done.